_sass/fix-analytics/card.scss

```diff
@@ -0,0 +1,62 @@
+.card {
+    border-radius: $border-radius;
+    display: flex;
+    flex-direction: column;
+    align-items: stretch;
+    background: white;
+    box-shadow: $overlay-shadow, 0 -4px 5px rgba(0,0,0,0.1) inset;
+}
+
+.card__header {
+    font-weight: 400;
+    margin-bottom: 1rem;
+    align-self: flex-start;
+    padding-top: 1.25rem;
+}
+
+.card__text {
+    padding: 0 1.25rem;
+
+    p {
+        margin-bottom: 1rem;
+    }
+}
+
+.card__actions {
+    display: flex;
+    flex-direction: row;
+    padding: 1rem 1.25rem;
+    margin-top: 1rem;
+}
+
+.card--dark-shiny {

```

Comment:
hmm, for at ungå en modifier til en modifier (dark og shiny) så kunne vi måske sige at det her bare er vores `card--dark`, eller splittet det op så vi har en `--dark` og en `--shiny` som virker uafhængig af hinanden. Jeg hælder nok mest til det første :).